addons/im_livechat/static/src/components/discuss_sidebar/discuss_sidebar.js

```diff
@@ -23,6 +23,7 @@ patch(components.DiscussSidebar, 'im_livechat/static/src/components/discuss_side
             .all(thread =>
                 thread.channel_type === 'livechat' &&
                 thread.isPinned &&
+                thread.messages.length &&
```

Comment:
This is too restrictive, could you please explain what is the issue? Why for livechat with guest the channel does not show, but for livechat with oneself it does show?

I guess the channel partner is created in both cases, but when the guest is created it, `is_pinned` is true only for the guest, while when it is created by oneself, `is_pinned` is true for oneself?

I think the proper fix should be along those lines, either the channel should not be created until there is a message, or that the channel partner `is_pinned` should be `false` when the livechat is initially created.

Reply:
Hello @seb-odoo 

when we try to do live chat from the visitor side, it will create two partner channels one for the visitor and another for admin.
In the visitor partner channel, we have 'is_pinned' true and for admin it is false.
that's why live chat is in an open state for visitor.

But, when we try to do live chat from the admin side, it will create only one partner channel.
and in that, it will consider admin as the visitor and sets 'is_pinned' true.
that's why having live chat opened both of the side (backend and frontend).

here, we can change is_pinned to false while sending the notification to admin when creating the LiveChat channel.

what is your view on this? Thanks

Reply:
Hello, thank you for the explanation, I understand the issue. But this is tricky and I'm not sure what is best, I need to think a bit more about a solution.
I'll get back to you when I have some time to think about it.

Reply:
Hello @seb-odoo 
I have done one fix for this issue.
we will set 'is_pinned' to false while sending live chat notification to the operator.
so we don't have live chat initially until the operator receives any message.
Can you please have a look if it looks the proper way or not?

Thanks